app/ui/client/lib/fileUpload.js

```diff
@@ -118,6 +119,13 @@ export const fileUpload = async (files, input, { rid, tmid }) => {
 		tmid = replies[0]._id;
 	}
 
+	const key = ['messagebox', rid, tmid].filter(Boolean).join('_');
+	let messageBoxText = Meteor._localStorage.getItem(key);
+
+	if (!messageBoxText) {
+		messageBoxText = '';
+	}
```

Comment:
you can use the double pipe operator here
`let messageBoxText = Meteor._localStorage.getItem(key) || '';`

Reply:
Ok, I'll do this